packages/guides/src/Nodes/Inline/AbstractLinkInlineNode.php

```diff
@@ -13,13 +13,32 @@
 
 namespace phpDocumentor\Guides\Nodes\Inline;
 
-abstract class AbstractLinkInlineNode extends InlineNode implements LinkInlineNode
+use Doctrine\Deprecations\Deprecation;
+use phpDocumentor\Guides\Nodes\InlineCompoundNode;
+
+abstract class AbstractLinkInlineNode extends InlineCompoundNode implements LinkInlineNode
```

Comment:
Just a fyi for next time: the change of parent class from `InlineNode` to `InlineCompoundNode` is a BC break for people that extend this base class in their code.

I've fixed this now in Symfony's alpha code, but it might be worth a notion in the release notes (and generally, I would advice to not introduce new deprecations in patch versions).